plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaConfig.java

```diff
@@ -152,4 +159,26 @@ public KafkaConfig setTimestampUpperBoundPushDownEnabled(boolean timestampUpperB
         this.timestampUpperBoundPushDownEnabled = timestampUpperBoundPushDownEnabled;
         return this;
     }
+
+    @NotNull
+    public List<@FileExists File> getResourceConfigFiles()
+    {
+        return resourceConfigFiles;
+    }
+
+    @Config("kafka.config.resources")
+    @ConfigDescription("Optional config files")
+    public KafkaConfig setResourceConfigFiles(String files)
+    {
+        this.resourceConfigFiles = SPLITTER.splitToList(files).stream()
+                .map(File::new)
+                .collect(toImmutableList());
+        return this;
+    }
+
+    public KafkaConfig setResourceConfigFiles(List<File> files)
```

Comment:
Are we using it only in testing code or ?

Reply:
Yes it's currently only used in the testing code. Do you prefer I remove it? I added it to make it consistent for getters/setters.

Reply:
Nice catch. Seems like it's a duplicate function. I'll remove it.